### ISSUE TITLE
Running nixes-install from SOQuartz SD image gives ` 'flakes' is disabled` error

### ISSUE THREAD

**Issue by msgilligan** [nabam/nixos-rockchip#26]
When I try to use the SoQuartz SD Image to install NixOS to NVMe, I get the following error:

```
error: experimental Nix feature 'flakes' is disabled; use '--extra-experimental-features flakes' to override
```

Using `sudo NIX_CONFIG="experimental-features = nix-command flakes" nixos-install` seems to work-around the problem, but when installing to NVMe from other NixOS installer images this isn't required. It seems like installers created with this image require flakes without flakes being enabled.

**Comment by Diamondtroller** [comment]
I don't think this is specific to the Quartz image, rather to sdcard with flake configs. Having same issue on rock pi 4c

**Comment by nabam** [comment]
@msgilligan do you know an elegant solution to this problem? If not I can just add instructions to the readme.

**Comment by msgilligan** [comment]
Not off the top of my head. I would say just add the instructions for now.
